src/Compilers/CSharp/Test/Syntax/Parsing/ExtensionParsingTests.cs

```diff
@@ -3658,32 +3169,7 @@ class C : extension<C>
         // {
         //    public abstract static explicit operator int(T i);
         // }
-        UsingTreeWithCSharpNext(text,
-            // (3,30): error CS1001: Identifier expected
-            //     static explicit extension<C>.operator int(C i)
-            Diagnostic(ErrorCode.ERR_IdentifierExpected, "<").WithLocation(3, 30),
-            // (3,33): error CS1514: { expected
-            //     static explicit extension<C>.operator int(C i)
-            Diagnostic(ErrorCode.ERR_LbraceExpected, ".").WithLocation(3, 33),
-            // (3,33): error CS1513: } expected
-            //     static explicit extension<C>.operator int(C i)
-            Diagnostic(ErrorCode.ERR_RbraceExpected, ".").WithLocation(3, 33),
-            // (3,33): error CS1519: Invalid token '.' in class, record, struct, or interface member declaration
-            //     static explicit extension<C>.operator int(C i)
-            Diagnostic(ErrorCode.ERR_InvalidMemberDecl, ".").WithArguments(".").WithLocation(3, 33),
-            // (3,46): error CS1519: Invalid token '(' in class, record, struct, or interface member declaration
-            //     static explicit extension<C>.operator int(C i)
-            Diagnostic(ErrorCode.ERR_InvalidMemberDecl, "(").WithArguments("(").WithLocation(3, 46),
-            // (3,50): error CS8124: Tuple must contain at least two elements.
-            //     static explicit extension<C>.operator int(C i)
-            Diagnostic(ErrorCode.ERR_TupleTooFewElements, ")").WithLocation(3, 50),
-            // (4,5): error CS1519: Invalid token '{' in class, record, struct, or interface member declaration
-            //     {
-            Diagnostic(ErrorCode.ERR_InvalidMemberDecl, "{").WithArguments("{").WithLocation(4, 5),
-            // (7,1): error CS1022: Type or namespace definition, or end-of-file expected
-            // }
-            Diagnostic(ErrorCode.ERR_EOFExpected, "}").WithLocation(7, 1)
-            );
+        UsingTreeWithCSharpNext(text);
```

Comment:
>UsingTreeWithCSharpNext(text); [](http://example.com/codeflow?start=8&length=30)

It looks like the comment above has been addressed

Reply:
done.